### ISSUE TITLE
Decide on learning decay rate and target games/generation numbers for 9s

### ISSUE THREAD

**Issue by amj** [tensorflow/minigo#53]

**Comment by amj** [comment]
Decisions:

 - Run for 4M steps at batch 256 (~1Bn positions)
 - Checkpoint every 2M positions for ~500 checkpoints
 - Target 5k games/generation.
 - Learning rate: set to .1, cut to .01 and .001 at 1M and 2M positions respectively.
